crates/bevy_pbr/src/render/mesh.wgsl

```diff
@@ -23,14 +27,24 @@ struct VertexOutput {
 [[group(2), binding(0)]]
 var<uniform> mesh: Mesh;
 
+#ifdef SKINNED
+[[group(3), binding(0)]]
+var<uniform> joint_matrices: SkinnedMesh;
+#import bevy_pbr::skinning
```

Comment:
Should this import be moved upper so that it's alongside other imports (in a separate `ifdef` block)? I almost missed it here...

Reply:
WGSL is a single-pass language, meaning the order in which symbols are introduced matters. The `joint_matrices` variable must be defined before the code in the #import. The resultant WGSL is not valid otherwise.

Reply:
Ah, I see! Makes total sense, thx!